static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -366,6 +366,7 @@ export class DeprecatedLine extends Component<Props, State> {
                   platform={this.props.platform ?? 'other'}
                   isHoverPreviewed={isHoverPreviewed}
                   meta={this.props.frameMeta}
+                  stackFramePath={this.props.frameSourceResolutionResults?.stackFramePath}
```

Comment:
we cannot use frameSourceResolutionResults because that data comes from the source map debugger modal data which will not contain stackFramePath if the event does not contain a release. I know this sounds fishy but the frameSourceResolutionResults was not intended to be used for this use case. Please use `event.exception.values[].raw_stacktrace.frames[].abs_path` instead.

Reply:
some kind of test would probably be good to verify this too